docs/source/extensions/overview_extensions.rst

```diff
@@ -17,34 +17,62 @@ Offically Supported Extensions
 The following extensions are officially supported and were developed as part
 of the core project:
 
-- :doc:`extension_covid19`: Make literature on COVID-19 available in ASReview.
-- :doc:`extension_visualization`: Plotting functionality for state files 
-  produced by ASReview.
-- :doc:`extension_wordcloud`: Create a visual impression of the contents of 
-  datasets via a wordcloud.
-- `ASReview-statistics <https://github.com/asreview/asreview-statistics>`__: 
-  Tool to give some basic properties of a dataset, such as number of papers, 
-  number of inclusions. 
-- `ASReview-vocab-extractor <https://github.com/asreview/asreview-extension-vocab-extractor>`__: 
-  This extension adds two feature extractors that extract vocabulary and 
-  vector matrices during simulation phases. Might one day be integrated to the 
-  core.
-- `ASReview-hyperopt <https://github.com/asreview/asreview-hyperopt>`__: 
-  Optimize the hyperparameters of the models in ASReview.
+
+Model extensions
+~~~~~~~~~~~~~~~~
```

Comment:
```suggestion

.. _extensions-official-model:

Model extensions
~~~~~~~~~~~~~~~~
```

I think something like this resolves the issue with labels and hyperlinks.

Reply:
Unfortunately it did not resolve the warning. Since we only have model extensions I suggest to remove the header.